pkg/resource/replication_group/testdata/test_suite.yaml

```diff
@@ -139,17 +138,15 @@ tests:
             - operation: DescribeEventsWithContext
               output_fixture: "events/read_many/rg_cmd_events.json"
         invoke: Delete
-        expect: # for the delete case we don't expect a new latest state or a non-nil error
-          latest_state: nil
+        expect:
```

Comment:
As there is no call to `sdkFind()` inside `sdkDelete()`, there is no need to mock `DescribeReplicationGroupsWithContext`, `ListAllowedNodeTypeModifications`  and `ListAllowedNodeTypeModifications` service APIs.